plugins/out_stackdriver/stackdriver.h

```diff
@@ -47,6 +47,12 @@
 #define FLB_SDS_RESOURCE_TYPE "global"
 
 #define OPERATION_FIELD_IN_JSON "logging.googleapis.com/operation"
+#define LOCAL_RESOURCE_ID_KEY "logging.googleapis.com/local_resource_id"
+#define LEN_LOCAL_RESOURCE_ID_KEY 40
+
+#define LEN_K8S_CONTAINER strlen("k8s_container")
```

Comment:
the common way to do this is like:

```
#define K8S_CONTAINER    "k8s_container"
```

then in the places that you need to get the string length you just do:

```
len = sizeof(K8S_CONTAINER) - 1;
```

> the value of  ```sizeof()``` is set at compile time

Reply:
updated.